src/repository.ts

```diff
@@ -190,6 +201,47 @@ abstract class Repository extends AsyncOptionalCreatable {
     return found;
   }
 
+  /**
+   * Analyzes the commits to see if any of them indicate that a new
+   * release should be published.
```

Comment:
Can we put in the comment that this is a workaround with a link to the issue on standard-version. I would like to remove this down the road if they update that library.